lib/i18n/de.json

```diff
@@ -289,7 +291,10 @@
         "accept": "Einladung annehmen",
         "reject": "Ablehnen",
         "warning_user_already_in_team": "Du kannst nur in einem Team Mitglied sein. Wenn du dem Team '${inviting_team}' beitrittst, verlässt du automatisch dein bisheriges Team '${current_team}'."
-      }
+      },
+      "select_team": "Team auswählen",
+      "select_team_hint_route": "Wähle das Team aus, dem du diese Route zuweisen möchtest. Beachte: Du kannst nur Teams aus deiner eigenen Gliederung auswählen",
```

Comment:
Isn't a state a 'Gliederung' as well? Should the text be more specific and talk about OV/KV? (I know the answer is "We just put the texts here", but this could confuse people...)

Reply:
Theoretically you are right, but I hold back with stuff that isn't a functional thing, and the text is already too much for the little space we have on the screen. And using abbreviations wouldn't help either as they introduce another term to explain. I leave it to the "experts" to phrase it how they want it.